db/data/queries/createTable.sql

```diff
@@ -1,3 +1,4 @@
+CREATE DATABASE `quickOrder`;
```

Comment:
**CREATE DATABASE** is in createDatabase.sql, there is no need to do it here

The reason i moved it to a separate **.sql** script back then is because when using docker you would get your database created automatically, so there is no need to create a database prior to creating a table